bindings/pyroot/pythonizations/test/rtensor.py

```diff
@@ -28,8 +29,10 @@ def test_dtypes(self):
         """
         Test adoption of numpy arrays with different data types
         """
-        #using long long on Linux does not work although it is same size as long
-        if (platform.system() == "Windows") :
+        # Using long long on 64 bit Linux does not work although it is
+        # same size as long
+        if ((platform.system() == "Windows") or
+            ((platform.system() == "Linux") and (sys.maxsize < 2**32))) :
```

Comment:
What if we scrap this extra check to alter the `cpptypes`, and directly define them to be what they are?

```c++
    cpptypes = ["int32_t", "int64_t", "uint32_t", "uint64_t", "float", "double"]
```
At least this is what I understand from the numpy docs:
https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.int64